Makefile

```diff
@@ -6,11 +6,15 @@ help :
 	@echo '  make test                  run unit tests'
 	@echo '  make lint                  run linter'
 	@echo '  make doc                   make documentation'
+	@echo '  make changelog             update changelog based on version'
 	@echo '  make dist                  make binary and source packages'
 	@echo '  make dist-check            verify binary and source packages'
 	@echo '  make upload                upload to PyPI'
 	@echo
 
+VERSION ?= $(shell python setup.py --version)
+REPO_URL = https://github.com/scimma/client_library/compare
```

Comment:
Should `REPO_URL` have a `/compare` at the end? This gives `Unreleased` URLs looking like `.../compare/compare/v[tag]` and a release URL like `.../compare/releases/tag/v[tag]`. Not sure if these are the desired URLs since I haven't used the web UI for comparisons, but I wanted to bring this up just in case.

Reply:
Nope, that's definitely a typo. Nice catch, I pushed a fix with this and rebased the branch with respect to `master` while I was at it.